packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -170,15 +173,30 @@ export async function handler(event: SQSEvent) {
         const payloadRaw = await s3Utils.getFileContentsAsString(s3BucketName, s3FileName);
         if (payloadRaw.includes("nonXMLBody")) {
           const msg = "XML document is unstructured CDA with nonXMLBody";
-          console.log(`${msg}, skipping...`);
+          log(`${msg}, skipping...`);
           capture.message(msg, {
             extra: { message, ...lambdaParams, context: lambdaName, fileName: s3FileName },
             level: "warning",
           });
           await ossApi.internal.notifyApi({ ...lambdaParams, status: "failed" }, log);
           continue;
         }
-        const payloadNoB64 = removeBase64PdfEntries(payloadRaw);
+        const { documentContents: payloadNoB64, b64Attachments } =
+          removeBase64PdfEntries(payloadRaw);
+
+        log(`Extracted ${b64Attachments.length} B64 attachments`);
+        if (b64Attachments.length) {
+          const fhirApi = makeFhirApi(cxId, fhirUrl);
+          processAttachments({
```

Comment:
need an await in lambdas I think... otherwise this may cutoff during execution...

Reply:
Oh.. I was actually leaving that async on purpose

Reply:
yea, but if it's async the lambda execution context might exit upon a return without this finishing and it'll just end immediately.

Reply:
cool, thx for letting me know! added